lingua_franca/lang/common_data_pt.py

```diff
@@ -116,3 +133,174 @@
     80: 'oitenta',
     90: 'noventa'
 }
+
+# split sentence parse separately and sum ( 2 and a half = 2 + 0.5 )
+_FRACTION_MARKER_PT = {"e"}
+# for non english speakers,  "X Y avos" means X / Y ,  Y must be > 10
+_SUFFIX_FRACTION_MARKER_PT = {"avos"}
+# mean you should sum the next number , equivalent to "and", i.e, "two thousand and one"
+_SUM_MARKER_PT = {"e"}
+# decimal marker ( 1 point 5 = 1 + 0.5)
+_DECIMAL_MARKER_PT = {"ponto", "virgula", "vírgula"}
+
+# negate next number (-2 = 0 - 2)
+_NEGATIVES_PT = {"menos"}
+# negate previous number, "2 negative" -> -2
+_NEGATIVE_SUFFIX_MARKER_PT = {"negativo", "negativos"}
+
+# _LONG_ORDINAL_PT, _LONG_SCALE_PT, _SHORT_SCALE_PT, _SHORT_ORDINAL_PT
+_LONG_SCALE_PT = OrderedDict([
+    (100, 'cem'),
+    (1000, 'mil'),
+    (1000000, 'milhão'),
+    (1e12, "bilião"),
+    (1e18, 'trilião'),
+    (1e24, "quadrilião"),
+    (1e30, "quintilião"),
+    (1e36, "sextilião"),
+    (1e42, "septilião"),
+    (1e48, "octilião"),
+    (1e54, "nonilião"),
+    (1e60, "decilião"),
+    (1e66, "undecilião"),
+    (1e72, "duodecilião"),
+    (1e78, "tredecilião"),
+    (1e84, "quattuordecilião"),
+    (1e90, "quinquadecilião"),
+    (1e96, "sedecilião"),
+    (1e102, "septendecilião"),
+    (1e108, "octodecilião"),
+    (1e114, "novendecilião"),
+    (1e120, "vigintilião"),
+    (1e306, "unquinquagintilião"),
+    (1e312, "duoquinquagintilião"),
+    (1e336, "sesquinquagintilião"),
+    (1e366, "unsexagintilião")
+])
+
+_SHORT_SCALE_PT = OrderedDict([
+    (100, 'hundred'),
```

Comment:
You might have missed some Portuguese translations here. Maybe it should be "cem" and "mil" :)

Reply:
this is a work in progress, i need to review all strings, i did a simple search and replace to "portuguesify" the names, the prefixes should be the same but i'm sure i missed some things

good catch! :)

Reply:
Sure, I just happened to notice, so I wrote a comment. :)